internal/internal_flags.go

```diff
@@ -44,7 +44,10 @@ const (
 	// a workflow task response's command set to order messages with respect to
 	// commands.
 	SDKFlagProtocolMessageCommand = 3
-	SDKFlagUnknown                = math.MaxUint32
+	// SDKPriorityUpdateHandling will cause update request to be handled before the main workflow method.
+	// It will also cause the SDK to immediately handle updates when a handler is registered.
+	SDKPriorityUpdateHandling = 4
```

Comment:
Updates are experimental, do we have to put this on everyone's history forever now to preserve compatibility for experimental users?

Reply:
We can deprecate these at some point, like when update goes GA or after a few releases

Reply:
Can we just release saying in some rare cases updates are incompatible with last release since they are experimental?

Reply:
I think we should be ok breaking workflows using experimental updates instead of adding a flag for everyone. With this change, you can never remove this flag enumerate. So we made something experimental so we could make changes, but we're not allowing ourselves to make changes, so the code is forever burdened.

Reply:
Yeah, that's an acceptable compromise. I understand the number 4 is now reserved forever, but ok. I personally prefer to break now, but your choice.

Reply:
Sorry to unresolve this...can we make a new issue or something so we don't forget to remove this flag some day instead of setting for everyone forever?

Reply:
https://github.com/temporalio/sdk-go/issues/1315